build.py

```diff
@@ -56,6 +56,9 @@
   from urllib.parse import urlencode
   from importlib import reload
 
+# Read package.json and extract the current Blockly version
```

Comment:
Prefer to end with a period.

"Complete sentences are recommended but not required. Complete sentences should use appropriate capitalization and punctuation."
https://google.github.io/styleguide/javascriptguide.xml?showone=Comments#Comments